pkg/webui/console/components/events/previews/previews.styl

```diff
@@ -14,5 +14,5 @@
 
 .plain-text
   font-family: $font-family-mono
-  font-size: $fs.xs
-  color: var(--c-text-neutral-heavy)
+  font-size: $fs.s
+  color: var(--text-neutral-heavy)
```

Comment:
Why remove the `--c`? Aren't all the color variables prefixed with `c`?

Reply:
The reason is that some new variables are assigned `pkg/webui/console/components/events/events.styl` to enable the dark layout. Since the new variable assignments are applied to the overall event container, all subtree definitions can use these variables.